tests/stories/LegacyMeta.stories.svelte

```diff
@@ -0,0 +1,9 @@
+<script context="module">
+  import { Meta, Story } from '@storybook/addon-svelte-csf';
+
+  import LegacyMeta from './LegacyMeta.svelte';
+</script>
+
```

Comment:
The legacy API has never worked without a `<Template>` call, so we should add one here, even though it technically works without it in v5.